src/vrapp/VolumeVisualizationApp.cpp

```diff
@@ -542,42 +539,40 @@ std::cout << "init vizapp " << std::endl;
     glClearDepth(1.0f);
     glDepthFunc(GL_LEQUAL);
     glClearColor(0.0, 0.0, 0.0, 1);
-     std::cout << "init vizapp end" << std::endl;
+    std::cout << "init vizapp end" << std::endl;
+    const float duration = 10.5f;
   }
 
-
-
-  
-  glMatrixMode(GL_MODELVIEW);
-  glLoadIdentity();
-  glLightfv(GL_LIGHT0, GL_POSITION, m_light_pos);
-
   if (m_vrVolumeApp)
   {
     if (m_vrVolumeApp->pending_models_to_load())
     {
       m_vrVolumeApp->load_mesh_model();
     }
-    
+
     m_vrVolumeApp->initialize_textures();
     m_vrVolumeApp->update_3D_ui();
     m_vrVolumeApp->update_trackBall_state();
-    m_vrVolumeApp->update_animation();
-    m_vrVolumeApp->set_render_count(0);
 
+    if (true)
```

Comment:
get rid of the if since it's hardcoded to `true`

Reply:
done